docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -28,13 +28,15 @@ Using Custom Galaxy in the Cloud with Terra, and AnVIL
 ------------------------------------------------------------------------
 Galaxy is also integrated into the Terra cloud workspace. This provides users the ability to readily create a custom Galaxy environment scaled to their needs or with additional tools installed. Additionally, the Terra workspace is able to access a rich data corpus hosted in the cloud, including many controlled access datasets, thus allowing users to conduct analyses with this data in a compliance-based environment.
 
-Unlike WDL and CWL, Galaxy workflows in the near term are created and modified from the Galaxy workflow editor (GUI), instead of a text editor.
-
 Create a basic Galaxy workflow
 ------------------------------
 
+Unlike WDL and CWL, Galaxy workflows are currently created and modified from the Galaxy workflow editor (GUI), instead of a text editor.
```

Comment:
Sentence may be a little weird if the user starts with Galaxy and doesn't know CWL or WDL.